classes/Utils.php

```diff
@@ -9060,6 +9060,28 @@ public function text_editor_config( $args = array() ) {
 		return wp_parse_args( $args, $default_args );
 	}
 
+	/**
+	 * Get config for profile bio editor.
+	 *
+	 * @since 2.2.4
+	 * 
+	 * @param string $textarea_name textarea name for post request.
+	 *
+	 * @return array
+	 */
+	public function get_profile_bio_editor_config( $textarea_name = 'tutor_profile_bio' ) {
```

Comment:
In the `Utils` class there is already a method that exists for a minimal text editor:
Please have a look on this method `text_editor_config`